app/models/concerns/team_private.rb

```diff
@@ -153,7 +153,7 @@ def get_dashboard_export_headers(ts, dashboard_type)
     # Hash to include top items as the header label depend on top_items size
     top_items = {}
     # tipline_dashboard columns
-    if dashboard_type == 'tipline_dashboard'
+    if dashboard_type == :tipline_dashboard
```

Comment:
@melsawy , in order to be more robust, I suggest you replace `dashboard_type` by `dashboard_type.to_sym`.